go.mod

```diff
@@ -3,14 +3,19 @@ module github.com/envoyproxy/gateway
 go 1.18
 
 require (
+	github.com/envoyproxy/go-control-plane v0.10.1
```

Comment:
Not sure why a go mod bump was needed when the API is not importing this pkg.

Reply:
its stale from the previous commit, fixed it here https://github.com/envoyproxy/gateway/pull/125